include/uart.h

```diff
@@ -49,6 +49,49 @@
 /*************************** Types Declarations *******************************/
 /******************************************************************************/
 
+/**
+ * @enum uart_size
+ * @brief UART character size (number of data bits) options.
+ */
+typedef enum uart_size {
```

Comment:
Linux coding style tends to avoid using typedefs

Reply:
Done.